examples/jenkins/pipeline/samplepipeline.yaml

```diff
@@ -0,0 +1,327 @@
+apiVersion: v1
```

Comment:
samplepipeline.json removed. I didn't find any tests that referred to it. Only the cluster up code, which I fixed up.